RPIPs/RPIP-59.md

```diff
@@ -1,90 +0,0 @@
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
-- There SHALL be an `express_queue`
-  - When adding a validator, users MAY place their deposit on the `express_queue` by spending one `express_queue_ticket`
-- When matching ETH from the deposit pool to queued deposits:
-  - First, ETH SHALL be matched to the oldest deposit in the `express_queue`; this is repeated `express_queue_rate` times
-  - Next, ETH SHALL be matched to the oldest deposit in the `standard_queue`
-  - This sequence SHALL be repeated indefinitely until both queues are empty
-    - If one queue is empty, those matches SHALL be skipped
-- Each node SHALL be provided `express_queue_tickets` equal to `express_queue_tickets_base_provision` (this includes newly created nodes)
-- Each node SHALL be provided additional `express_queue_tickets` equal to `(bonded ETH in legacy minipools)/4` (this will always be zero for newly created nodes)
-- The initial settings SHALL be:
-  - `express_queue_rate`: 2
-  - `express_queue_tickets_base_provision`: 2
-
-### Deposit mechanics specification
-- A node operator MUST take 2 actions to start a validator: `deposit` and `stake`
-
-#### `deposit` Transaction
-- `deposit` SHALL place the Node Operator ETH in the deposit pool (where it can be used in validators as needed) and place the validator in a queue as described [above](#deposit-queue-specification)
-- The following values for the validator SHALL be stored on chain:
-    - the public key of the validator
-    - the BLS signature over the public key, the withdrawal credentials (the megapool address of the node operator), and the amount (1 ETH) as required by the deposit contract
-    - the deposit message root as required by the deposit contract
-- The transaction SHALL validate the provided values and revert if they do not match
-- `deposit` SHALL assign deposits as described below 
-
-#### Assigning ETH from the Deposit Pool
-- As ETH enters the deposit pool, it SHALL be assigned to valdiators from the queue by sending 32 ETH to the associated megapool contract
-- The assignment SHALL execute the `Prestake` transaction, staking 1 ETH to the beacon chain using the values provided in the step above
-
-#### `stake` Transaction
-- `stake` SHALL revert unless at least `scrub_period` time has passed since ETH was assigned to the validator, to allow for validating the prestake
-- If the beacon chain stake is invalid, the validator SHALL be scrubbed 
-- `stake` SHALL stake the remaining 31 ETH to the beacon chain to make a complete validator
-- If `stake` is not called within `time_before_dissolve` after the ETH was assigned, the validator SHALL be dissolved, returning the user balance to the deposit pool
-
-#### Exiting Queue
-- Until ETH is assigned to a validator, it SHALL be possible to exit the queue and receive ETH `credit` for it
-
-#### Social Assignments
-- The deposit.assign.socialised.maximum setting SHALL be set to 0
-
-
-#### Initial Settings
```

Comment:
Who can change, when, etc.

Reply:
All pDAO settings are controlled by the pDAO by vote. Anything other than that needs to be explicitly specified.

Reply:
Right, but do we even say these are pDAO settings in here?

Reply:
There are no other setting types. There is only one thing we call settings. They can be changed by the pDAO or oDAO. They can be changed by the Security Council only if that right is explicitly given for that setting.

Reply:
There are oDAO settings and pDAO settings. Separately from the security council point here, I don't think we've said anywhere that these are pDAO settings. Like, the word 'pDAO' does not appear here, does it?

Reply:
Crap, you're right. We do have a couple small sets of oDAO settings that the pDAO cannot change. The oDAO can change the pDAO settings (indirect or not, they can do it). I agree we should be saying "pDAO setting" in a thousand places. I will not be fixing that for the PR I'm working on right now.

Reply:
So we do need to explicitly state that? Why not just explicitly state when it's a oDAO setting and pDAO setting can be the default assumption...